visualization/codemirror/sourceCode.js

```diff
@@ -97,7 +97,9 @@ var sourceCode = (function() {
 	 * Clear all three arrays
 	 */
 	function emptyArrays() {
-		currentAsatWarnings, currentCatWarnings, currentMessageWarnings = [];
+		currentAsatWarnings = [];
```

Comment:
The method using the comma caused a bug, that's why I changed back to the old way.